flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -107,6 +107,10 @@ void testExplainAsDot() {}
     void testStringAgg() {}
     // END
 
+    @Disabled
```

Comment:
Would you like to help me understand the reason to add this test and disabled?

Reply:
If we are talking about this concrete case then ideally it should be fixed before merging this PR
and I'm trying to find a way how to do that, similar issue was highlighted in Calcite related PR

If we are talking about in general then it is a normal way to turn off Calcite parser tests if for some reason parser features for Calcite and Flink are different since we can not override these tests (Calcite does not allow it)

Reply:
no need for this anymore, reverted these lines